python/GangaLHCb/Lib/Applications/GaudiExecUtils.py

```diff
@@ -99,7 +99,6 @@ def prepare_cmake_app(myApp, myVer, myPath='$HOME/cmtuser', myGetpack=None):
     full_path = expandfilename(myPath, True)
     if not path.exists(full_path):
         makedirs(full_path)
-        chdir(full_path)
```

Comment:
`getpack` seems to be called using `subprocess.Popen` with the current working directory set to `dev_dir` so that should be fine.

Here the directory was only changed to `$HOME/cmtuser` so it's still the wrong place for calling `getpack/lb-.*`.